cmd/translatesfx/README.md

```diff
@@ -1,8 +1,8 @@
-# SignalFx Smart Agent Configuration Translation Tool (Experimental)
+# SignalFx Smart Agent Configuration Translation Tool (Alpha)
 
 This package provides a command-line tool, `translatesfx`, that translates a 
 SignalFx Smart Agent configuration file into a configuration that can be
-used by an OpenTelemetry Collector.
+used by a Splunk Otel Collector.
```

Comment:
Suggested change: Splunk Distribution of OpenTelemetry Collector (if referring to the distribution) or Splunk OpenTelemetry Connector (if referring to a package)